app/design/adminhtml/Magento/backend/web/css/styles-old.less

```diff
@@ -1366,7 +1366,7 @@
         font-size: 14px;
         font-weight: 600;
         line-height: 3.2rem;
-        padding: 0 30px 0 0;
+        padding: 0 30px 0 30px;
```

Comment:
Hi @sudhanshu-bajaj. Thanks for collaboration. This changes will affect all labels in each fieldset. Are you check how it will look in other usages of this styles?

Reply:
@VladimirZaets Thanks for you feedback. I went through all sections by searching this class but it is not affecting any other layout. Please let me know if you found any affected area because of it.

Reply:
Hey @VladimirZaets , Is there anything we need action ?